client/state/account-recovery/reducer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+} from 'state/action-types';
+
+const reducer = ( state = {}, action ) => {
```

Comment:
If the settings are broken out as in the comment above, this can even be simplified to:

```js
export const reducer = createReducer( {}, {
    [ ACCOUNT_RECOVERY_FETCH_SUCCESS ]: ( { email, email_validated, phone, phone_validated } ) => ( {
        email,
        emailValidated: email_validated,
        phone,
        phoneValidated: phone_validated,
    } )
} );
```